variables.es

```diff
@@ -0,0 +1,35 @@
+import pkg from "./package.json"
+
+// shell has some limitation :)
+const __PROD__ = JSON.parse(Boolean(process.env.__PROD__))
+
+const variables = {
+  __VERSION__: process.env.__VERSION__ || pkg.version,
```

Comment:
for us the package .version has always be 0.0.0 so not sure what is the value added.
For lib I understand, for project like this (or some other that are CI based) I don't think a "semver" version is appropriate. Reminder: CI means continuous integration, so I am not sure a number to bump by hand is appropriate for web stuff.